### PR TITLE
feat: create sub command delete from personal token

### PR DESCRIPTION
**Pull request by maxwelbm**
- create sub command delete from personal token

### CODE REVIEW

File: pkg/cmd/personal-token/delete/detele.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	var id string
+
+	deleteCmd := &cobra.Command{
+		Use:           msg.DeleteUsage,
+		Short:         msg.DeleteShortDescription,
+		Long:          msg.DeleteLongDescription,
+		SilenceErrors: true,
+		SilenceUsage:  true,
+		Example: heredoc.Doc(`
+		$ azioncli personal_token delete --id 7a187044-4a00-4a4a-93ed-d230900421f3
```
Comment (gabriel-azion) [inline pkg/cmd/personal-token/delete/detele.go:24]:
```suggestion
		$ azion personal_token delete --id 7a187044-4a00-4a4a-93ed-d230900421f3
```

File: pkg/cmd/personal-token/delete/detele.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+		$ azioncli personal_token delete -i 7a187044-4a00-4a4a-93ed-d230900421f3
```
Comment (gabriel-azion) [inline pkg/cmd/personal-token/delete/detele.go:25]:
```suggestion
		$ azion personal_token delete -i 7a187044-4a00-4a4a-93ed-d230900421f3
```

**Review by gabriel-azion — DISMISSED** [review]

File: pkg/cmd/personal-token/delete/detele.go (hunk 2 above)
Comment (gabriel-azion) [reply pkg/cmd/personal-token/delete/detele.go:25]: I have a question. Wouldn't the shortened flags be used only for global vars?

Comment (maxwelbm) [reply pkg/cmd/personal-token/delete/detele.go:25]: As far as I know, shorthand are for IDs too

Comment (PatrickMenoti) [reply pkg/cmd/personal-token/delete/detele.go]: It's worth checking with Tiago, if that idea will continue in azion 1.0

**Review by gabriel-azion — DISMISSED** [review]

**Review by PatrickMenoti — APPROVED** [review]

**Review by ftoldato — APPROVED** [review]

**Pull request merged**